Libplanet/Net/Swarm.TrustedStateCompleter.cs

```diff
@@ -1,155 +0,0 @@
-                            request,
-                            timeout: Options.RecentStateRecvTimeout,
-                            cancellationToken: cancellationToken
-                        );
-                    }
-                    catch (TimeoutException e)
-                    {
-                        string msg = "Failed to receive the block {BlockHash}'s states from " +
-                            "a trusted peer {Peer}: " + e;
-                        _logger.Error(e, msg, blockHash, peer);
-                        continue;
-                    }
-
-                    if (reply is BlockStates blockStates)
-                    {
-                        if (!blockStates.BlockHash.Equals(blockHash))
-                        {
-                            string msg =
-                                $"Expected a {nameof(BlockStates)} message for the block " +
-                                "{RequestedBlockHash}, but received a wrong reply instead from " +
-                                "a trusted peer {Peer}: {RepliedBlockHash}";
-                            _logger.Error(msg, blockHash, peer, blockStates.BlockHash);
-                            continue;
-                        }
-                        else if (blockStates.States is null)
-                        {
-                            string msg =
-                                "A trusted peer {Peer} does not have the block {BlockHash}'s " +
-                                "states.";
-                            _logger.Error(msg, peer, blockHash);
-                            continue;
-                        }
-
-                        blockStatesStore.SetBlockStates(blockStates.BlockHash, blockStates.States);
-                    }
-                    else
-                    {
-                        string msg =
-                            $"Expected a {nameof(BlockStates)} message, but received a wrong " +
-                            "reply instead from a trusted peer {Peer}: {ReplyMessage}";
-                        _logger.Error(msg, peer, reply);
-                    }
-                }
-            }
-
-            return new StateCompleterSet<T>
-            {
-                StateCompleter = (blockChain, blockHash, address) =>
-                {
-                    FillTrustedBlockStates(blockHash).Wait(cancellationToken);
-                    return blockChain.GetState(address, blockHash, fallback.StateCompleter);
-                },
-                FungibleAssetStateCompleter = (blockChain, blockHash, address, currency) =>
-                {
-                    FillTrustedBlockStates(blockHash).Wait(cancellationToken);
-                    return blockChain.GetBalance(
-                        address,
-                        currency,
-                        blockHash,
-                        fallback.FungibleAssetStateCompleter
-                    );
-                },
-            };
-        }
-    }
-}
```

Comment:
`Libplanet/Net/Swarm.TrustedStateCompleter.cs` was removed because `Swarm<T>` hasn't supported the feature and `TrieStateStore` will not need the feature.